docs/configuration/configuration.md

```diff
@@ -238,6 +238,10 @@ hetzner_sd_configs:
 kubernetes_sd_configs:
   [ - <kubernetes_sd_config> ... ]
 
+# List of Lightsail service discovery configurations.
+lightsail_sd_configs:
```

Comment:
This also needs to go to the alerting SD section

Reply:
https://github.com/prometheus/prometheus/pull/8693/commits/ff78f97d5139e5c9f4b171c7536c9e3ec11d840c